### ISSUE TITLE
localhost:3000 not responding

### ISSUE THREAD

**Issue by MichaTausKamR** [flyntwp/flynt#208]
Hey guys, I installed your theme to my local wordpress but I can't seem to run it properly. I get to position 5. "Run npm run start and start developing. Your local server is available at localhost:3000."

I run the command, Windows Firewall is asking for permission, but after that, I get a message from my browser, that the webserver is not responding. I can open my wordpress installation and see that the theme is installed correctly. But I can't access the localhost:3000.

I tried googling but that didn't help yet. Obviously. ;) So it would be very nice to hear from you guys.

Regards,
Michael

**Comment by steffenbew** [comment]
Hi Michael, happy to hear you're giving Flynt a try! 
Did you follow step 2 of the install guide and adjust the host variable to match your local url?
`Change the host variable in flynt/build-config.js to match your host URL: const host = 'your-project.test'`

**Comment by MichaTausKamR** [comment]
Thanks for the fast response! Yes, I changed it to localhost. When I do netstat, I can see that node.exe listens on port 3000.Am 04.12.2019 17:23 schrieb Steffen Bewersdorff <notifications@github.com>:Hi Michael, happy to hear you're giving Flynt a try!
Did you follow step 2 of the install guide and adjust the host variable to match your local url?
Change the host variable in flynt/build-config.js to match your host URL: const host = 'your-project.test'

**Comment by domtra** [comment]
Hi Michael,
so your actual WordPress installation is accessible via `http://localhost`? Then it should be `const host = 'localhost'`. If you did set up a VirtualHost under some other domain, please adjust the `host` variable accordingly.
It could also be, that a browersync setting is preventing `localhost:3000` to work correctly. Please try setting `https` to `false` in `build-config.js` under `browserSync`. https://github.com/flyntwp/flynt/blob/master/build-config.js#L51

Hope this helps.

**Comment by MichaTausKamR** [comment]
Hello domtra, yes my WordPress runs on http://localhost. I tried that again and could even work with phpmyadmin in xampp.

Anyway, I tried your changes and it did something. Instead of giving me the error instantly, it now waits about a minute and then throws up an error "Firefox can't establish a connection to localhost."

:(

I only have Windows Defender and the Windows Firewall as security.

Regards
